src/backend/index.ts

```diff
@@ -116,16 +116,15 @@ const setup = async () => {
   app.use(media.proxyMiddleware)
 
   // Successful LTI resource link launch
-  // @ts-expect-error @types/ltijs
-  ltijs.onConnect(async (idToken, req, res) => {
+  ltijs.onConnect((idToken, req, res) => {
     if (
-      idToken.iss ===
-        'https://repository.staging.cloud.schulcampus-rlp.de/edu-sharing' ||
+      (config.ENVIRONMENT === 'staging' &&
+        idToken.iss === config.EDUSHARING_RLP_URL) ||
       idToken.iss === 'http://localhost:8100/edu-sharing'
     ) {
-      await onConnectEdusharing(idToken, req, res)
+      void onConnectEdusharing(idToken, req, res)
     } else {
-      onConnectDefault(idToken, req, res)
+      void onConnectDefault(idToken, req, res)
     }
```

Comment:
Why should `onConnectEdusharing` be limited to only the staging environment? 
Plus, I think we can simplify this check using 
```javascript 
if(idToken.iss.includes('edu-sharing')) { onConnectEdusharing(...
```
(at least until we get an edu-sharing instance that does not have 'edu-sharing' in the iss but so far every instance does)